content/javascript/concepts/storage/terms/length.md

```diff
@@ -0,0 +1,22 @@
+# .length()
+
+.length() is used on objects such as strings and arrays to return the length of the object.
+The .length() function does vary when used on string vs arrays. 
+
+## Syntax
+length is called on a string through the following syntax and will output the number of characters of the string. 
+```
+const pet = dog;
+console.log(pet.length);
+///returns 3;
+```
+## Syntax 
+
+length is called on an array through the following syntax.
+
+```
+const array = [1, 2, 3, 4];
+console.log(array.length);
+/// returns 4
+```
+Remember arrays are 0 Indexed but .length will count the 0 as 1 in it's count. 
```

Comment:
Why not remove all this and add this code block in the above syntax itself. After that create a example section for the code blocks